modules/virt-define-guest-agent-ping-probe.adoc

```diff
@@ -24,15 +24,23 @@ include::snippets/technology-preview.adoc[]
 .Sample guest agent ping probe
 [source,yaml]
 ----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  annotations:
+  name: fedora-vm
+  namespace: openshift-cnv
```

Comment:
```suggestion
  namespace: example-namespace
```

IIRC, `openshift-cnv` is reserved for CNV components/the HyperConverged Operator, and shouldn't be used for VMs